app/src/main/java/com/hcodez/android/ui/MainActivity.java

```diff
@@ -1,17 +1,50 @@
 package com.hcodez.android.ui;
```

Comment:
the main activity should the activity around which the app revolves around, this is a LoginActivity